rotatingtext/src/main/java/com/sdsmdg/harjot/rotatingtext/models/Rotatable.java

```diff
@@ -34,28 +39,51 @@
 
     private int FPS = 60;
 
+    private int index = 0;
+
     public Rotatable(int updateDuration, String... text) {
         this.updateDuration = updateDuration;
         this.text = text;
         currentWordNumber = -1;
     }
 
     public Rotatable(int color, int updateDuration, String... text) {
-        this.color = color;
+        colorList.add(color);
```

Comment:
This can be related to the review I gave above on the changes made by you in `RotatingTextSwitcher` file. Rather than deliberately adding the color to a list, try making a separate rotatable wherein the user can pass his `colorList` if he needs to do so.

Reply:
As the constructor for each rotatable gets called only once ... the contents of the list(colorList) and the variable(color) at that point are the same. In this way the functions in the original code are not disturbed.

Reply:
I know that they are the same initially but what we want is a new rotatable wherein the user can directly pass his color array/list in order of the colors he wants his different rotating texts to appear in adding them all at once, rather than step by step addition of colors to list. This will be much more easier for the user and will ensure accurate allotment of colors. You can remove the spinner and just show that  colors are changing on one of the two rotatables in the app. Get this done. @Anu-123-gif

Reply:
I have made the necessary changes thank you for your feedback.